README.md

```diff
@@ -1 +1,134 @@
-# Microverse-Portfolio-Project
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#hello microverse)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#html-css)
+    - [Key Features](#home-page)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#amen-musingarimi)
+- [🔭 Future Features](#the other pages)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#microverse)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Microverse-Portfolio-Project] <a name="Portfolio Website"></a>
+
+**[Microverse-Portfolio-Project]** is a portifolio website that shows information about me and the work that I have done
+
+## 🛠 Built With <a name="html and css"></a>
+
+### Tech Stack <a name="html and css"></a>
+
+<details>
+  <summary>A test project</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>N/A</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[home_page]**
+- **[nav_bar]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need any code editor, for example Visual Studio.
+
+### Setup
+
+Clone this repository to your desired folder
+
+### Run tests
+
+To run tests, run the following command: npx hint . and npx stylelint "\*_/_.{css,scss}"
+
+### Deployment
+
+You can deploy this project using Heroku
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Amen-Musingarimi)
+- Twitter: [@twitterhandle](https://twitter.com/Amen-Musingarimi)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/atmusingarimi/)
+
```

Comment:
- [OPTIONAL] Good job adding the links to your social media pages :clap: ,I think in order to make it look more professional, it would be better to replace the handles examples with your own too, for example changing @githubhandle into @Amen-Musingarimi .